Servers/Tomcat v9.0 Server at localhost-config/catalina.properties

```diff
@@ -0,0 +1,206 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Same here. Do you really change some default configuration of cataline properties?